src/main/resources/messages/LanguageServerBundle.properties

```diff
@@ -153,3 +153,8 @@ lsp.refactor.rename.symbol.handler.title=Rename LSP Symbol
 lsp.refactor.rename.cannot.be.renamed.error=The element can't be renamed.
 lsp.refactor.rename.prepare.progress.title=Prepare renaming for ''{0}'' file at {1} offset...
 lsp.refactor.rename.progress.title=Renaming ''{0}'' file with ''{1}'' new name...
+
+# LSP Command
+lsp.command.error.title=Cannot execute ''{0}'' command.
+lsp.command.error.with.ls.content=Unable to execute the nonexistent command ''{0}''; it is not defined by the language server ''{1}''. Please register the command via an IntelliJ plugin.
+lsp.command.error.without.ls.content=Unable to execute the nonexistent command ''{0}''. Please register the command via an IntelliJ plugin.
```

Comment:
We need a link to the doc on how to contribute plugins
```
 Missing ''{0}'' command! It needs to be contributed by an [IntelliJ plugin](https://github.com/redhat-developer/lsp4ij/blob/main/docs/DeveloperGuide.md#lsp-commands).
 ```

Reply:
fixed